src/coord/src/catalog/builtin.rs

```diff
@@ -1630,7 +1630,19 @@ WHERE false",
     needs_logs: false,
 };
 
-// Next id BuiltinView: 5037
+pub const PG_ROLES: BuiltinView = BuiltinView {
+    name: "pg_roles",
+    schema: PG_CATALOG_SCHEMA,
+    sql: "CREATE VIEW pg_roles AS SELECT
+    name::pg_catalog.text AS rolname,
```

Comment:
You can remove the cast here and on `oid` since they are already the correct type. See https://github.com/MaterializeInc/materialize/blob/202e1fa9178bb550f8b9c951ef2bf34d2152b253/src/coord/src/catalog/builtin.rs#L1230 for example.